wallet/disksync.go

```diff
@@ -23,7 +23,7 @@ import (
 
 // checkCreateDir checks that the path exists and is a directory.
 // If path does not exist, it is created.
-func checkCreateDir(path string) error {
+func CheckCreateDir(path string) error {
```

Comment:
I not sure if this should be exported from the package.  It's basic operations that I'm not really sure make much sense to be exposed from this package.